.github/workflows/combine-dependabot-prs.yml

```diff
@@ -0,0 +1,218 @@
+name: 'Combine Dependabot PRs'
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  combine-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        id: create-combined-pr
+        name: Create Combined PR
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Might be a dumb question, but where's this secret defined? (And why do we need a custom token instead of the default generated one?)

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reply:
Oh right, it _is_ the default one

You can remove this line then, the `github-script` action uses it by default